Examples/NN/example_MLP_DLR_airfoil.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python
+#
+# Example of MLP.
+#
+# Last revision: 09/10/2024
+
+import os, numpy as np, torch, optuna
+import pyLOM
+
+
+def load_dataset(fname,inputs_scaler,outputs_scaler):
+    '''
+    Auxiliary function to load a dataset into a pyLOM
+    NN dataset
+    '''
+    d  = pyLOM.Dataset.load(fname)
+    td = pyLOM.NN.Dataset(
+        variables_out  = (d["CP"],), 
+        variables_in   = d.xyz,
+        # to have each Mach and AoA pair just once. 
+        # To have all possible combinations, use [d.get_variable('AoA'), d.get_variable("Mach")]
+        parameters     = [[*zip(d.get_variable('AoA'), d.get_variable('Mach'))]], 
+        inputs_scaler  = inputs_scaler,
+        outputs_scaler = outputs_scaler,
+    )
+    return d, td
+
+def print_dset_stats(name,td):
+    '''
+    Auxiliary function to print the statistics
+    of a NN dataset
+    '''
+    x, y = td[:]
+    pyLOM.pprint(0,f'name={name} ({len(td)}), x ({x.shape}) = [{x.min(dim=0)},{x.max(dim=0)}], y ({y.shape}) = [{y.min(dim=0)},{y.max(dim=0)}]')
+
+
+## Set device
+device = pyLOM.NN.select_device("cpu") # Force CPU for this example, if left in blank it will automatically select the device
+
+
+## Load datasets and set up the results output
+BASEDIR = './DATA'
+CASESTR = 'NRL7301'
+RESUDIR = 'MLP_DLR_airfoil'
+pyLOM.NN.create_results_folder(RESUDIR)
+
+scaler     = pyLOM.NN.MinMaxScaler()
```

Comment:
@DavidRamosArchilla I get this error:
```
Traceback (most recent call last):
  File "/home/arnaumiro/Documents/LowOrder/Examples/NN/example_MLP_DLR_airfoil.py", line 48, in <module>
    _,td_train = load_dataset(os.path.join(BASEDIR,f'{CASESTR}_TRAIN.h5'),scaler,scaler)
  File "/home/arnaumiro/Documents/LowOrder/Examples/NN/example_MLP_DLR_airfoil.py", line 17, in load_dataset
    td = pyLOM.NN.Dataset(
  File "/home/arnaumiro/Documents/LowOrder/pyLOM/NN/utils.py", line 168, in __init__
    self.variables_in = inputs_scaler.transform([self.variables_in[:, i] for i in range(self.variables_in.shape[1])])
  File "/home/arnaumiro/Documents/LowOrder/pyLOM/NN/utils.py", line 70, in transform
    min_val = self.variable_scaling_params[i]["min"]
IndexError: list index out of range
```
Can you confirm?

Reply:
that's because the same scaler is being passed for the inputs and for the outputs when loading the datasets:
load_dataset(os.path.join(BASEDIR,f'{CASESTR}_TRAIN.h5'),scaler,scaler)

i will make a commit with the changes

Reply:
Ok, so that's python being python with the soft copies. I understand.